ui/components/app/assets/asset-list/asset-list.tsx

```diff
@@ -195,25 +197,14 @@ const AssetList = ({ onClickAsset, showTokensLinks }: AssetListProps) => {
         title={nativeCurrency}
         // The primary and secondary currencies are subject to change based on the user's settings
         // TODO: rename this primary/secondary concept here to be more intuitive, regardless of setting
-        primary={
-          showSecondaryCurrency(
-            isOriginalNativeSymbol,
-            useNativeCurrencyAsPrimaryCurrency,
-          )
-            ? secondaryCurrencyDisplay
-            : undefined
-        }
+        primary={isOriginalNativeSymbol ? secondaryCurrencyDisplay : undefined}
         tokenSymbol={
-          useNativeCurrencyAsPrimaryCurrency
+          showNativeTokenAsMainBalance
```

Comment:
I'm a little confused why this is here.  Shouldn't this setting only affect the main balance at the top?  Why would it affect an asset in the asset list?

Reply:
It's not affecting the whole tokenList; this `TokenListItem` is the native currency; which is affected by the setting;

When we are displaying the aggregated balance in Fiat; we wanted the native token to display also fiat value first;
Similary, if the setting is ON and we are displaying crypto value; we wanted to display the crypto value as primary value.